_blinkr/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ruby:2.1.10
+MAINTAINER ihamilto@redhat.com
+RUN apt-get update
+RUN apt-get install sudo
+
+RUN wget https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-linux-x86_64.tar.bz2
+RUN tar xvjf phantomjs-2.1.1-linux-x86_64.tar.bz2 -C /usr/local/share/
+RUN sudo ln -sf /usr/local/share/phantomjs-2.1.1-linux-x86_64/bin/phantomjs /usr/local/bin
```

Comment:
If you can do all of these in one line by `&&` like you would on the command line it would be better.